app/models/katello/content_view_puppet_environment.rb

```diff
@@ -112,6 +112,9 @@ def ignore_global_proxy
       false
     end
 
+    def ignorable_content
```

Comment:
It would be good to have a comment on why CVPE needs this empty method here (not a blocker, but just a suggestion)